R/report.stanreg.R

```diff
@@ -18,7 +18,7 @@
 #'   as.data.frame(r)
 #'   summary(as.data.frame(r))
 #' }
-#' @include report.lm.R
+#' @include report.lm.R report.lme4.R
```

Comment:
I'm not sure that's necessary? 🤔

Reply:
But `report_random.merMod` is in `report.lme4.R` which is used in `report_random.stanreg`, isn't it?